app/lib/features/pins/pages/pins_page.dart

```diff
@@ -67,9 +71,10 @@ class PinsPage extends ConsumerWidget {
               }
               return SliverGrid.builder(
                 itemCount: pins.length,
-                gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+                gridDelegate:
+                    SliverGridDelegateWithFixedCrossAxisCountAndFixedHeight(
                   crossAxisCount: max(1, min(widthCount, minCount)),
-                  childAspectRatio: 6,
+                  height: MediaQuery.of(context).size.height * 0.1,
```

Comment:
I think this causes some ugly design, now:

![grafik](https://github.com/acterglobal/a3/assets/40496/792d5fce-265c-4b51-8fb1-ea2f1ddd7e9e)

Reply:
Let me take a look at it and do quick fix!

Reply:
Increased the height of card to contain chip: ``` height: MediaQuery.of(context).size.height * 0.15,```